core/blockchain.go

```diff
@@ -80,6 +80,8 @@ var (
 
 	errInsertionInterrupted = errors.New("insertion is interrupted")
 	errChainStopped         = errors.New("blockchain is stopped")
+
+	lastWrite uint64
```

Comment:
Ah. now I see. So it was actually a 'floating global' in the original code. That's pretty ugly. It was better when you put it on the blockchain object. 

I had thought it was some loop-locla closure-variable.